### PR TITLE
Add dropdown menus

### PR DESCRIPTION
**Pull request by scribblemaniac**
Display dropdown menus on hover for the Community and Docs menu items.

Demo

### CODE REVIEW

File: _sass/blog/_layout.scss [hunk 1]
```diff
@@ -103,7 +103,58 @@
...
+        display: block;
+
+        visibility: hidden;
+        transform-origin: top;
+        -webkit-transform-origin: top;
+        transform: scaleY(0);
+        -webkit-transform: scaleY(0);
+        -ms-transform: scaleY(0);
+
+        transition: transform 01.25s ease;
```
Comment (J5lx) [inline _sass/blog/_layout.scss]: Shouldn’t this be .25, too? Or 1.25 on the prefixed lines?

Comment (scribblemaniac) [reply _sass/blog/_layout.scss]: Oops, that was for testing, it should be 0.25 too. Will fix soon, thanks!

**Comment by scribblemaniac** [comment]
@J5lx True, animations can be annoying, but I think what I've included is fairly unobtrusive (at 0.25s). People can try it out before merging this by visiting my fork: https://scribblemaniac.github.io/pencil2d.github.io/.

You will find that Jekyll, whether you are using it locally or on a fork, will work much better with the code from #12. All you need to do is adjust the url and baseurl in _config.yml and everything *should* work. Github produces a warning as well if you don't delete the CNAME file. You just have to be careful not to merge those changes.

**Comment by J5lx** [comment]
Ah yeah, that feels pretty usable to me. I think there’s also a way to have the collapsing animated, too, maybe I can look into that tomorrow or so. Also, it’s not that I have problems with Jekyll, it’s just that I like to install such software through the system package manager, and because Jekyll requires a whole bunch of dependencies from the AUR that’ll take a short while since I recently replaced my disk and took the opportunity to do a new setup from scratch.

**Comment by chchwy** [comment]
Anyway let's merge it and see what people say.

**Comment by J5lx** [comment]
I think this distinction between community page and the footer icons is not entirely clear, particularly since the community page includes not just sites that are entirely “user-run” but also “official” resources that are managed by us, like the forum and the Discord server. This is probably best exemplified by the Tumblr link which shows a screenshot of our own official Tumblr blog, but directs users to the pencil2d tag page. At best, the line is pretty blurry there, and before you mentioned it here, I had no idea there was an intentional difference between the two locations. Also, if we’re going to add to the footer icons, we should probably put in our YouTube channel there as well.

That said, I do like the Godot page. However, it seems to me that it does not have any distinction between “official” and ”community” resources like ours (i.e. they also got GitHub, Twitter, their forum etc. on there). It also does a slightly better job at putting everything in context, not just, like, “here’s a great place to meet other users, but also here, here, here and here” like our page.

> And we have a official tumblr blog, it should go to the footer

As I sorta mentioned this in chat yesterday, I could also hook up a tumblr blog at some-subdomain.pencil2d.org if you want, just let me know in that case.

**Comment by chchwy** [comment]
Let me explain why I am not a fan of dropdown list.

A principle of dropdown list, the longer the list is, the less useful it will be. A dropdown with 3 options, yes, it's clear and efficeint. A dropdown with 15 options, it's terrible. You must squint your eyes, slowly move the cursor between small clickable areas, and then carefully select an item. Things are getting worse if there are sub-menus because the cursor has to move in a narrow tunnel to "navigate" to the next level. Luckily we don't have a sub-menu. Currently there are 6 items in the community menu, it's ok. If we are going to add more sites like twitter, github, etc. I would say let's stop using dropdowns.

Second reason, it's not very mobile friendly. Think about how users browse the website on an iPad. How can a user "hovers over" the the menu on a touch screen by his fingers? With a mouse we can accurately control the cursor in dropdowns. What about fingers? That's why dropdowns are a common UI component on desktop but we rarely see it in mobile apps. In mobile apps, a long list is usually designed to be a individual page (thiking about the Settings of Android and iOS).

Third, If I want to show a user all the pencil2d groups, I would prefer sending him/her a link "https://www.pencil2d.org/community" rather than tell him/her please go to the home page and take a look at the community menu. Maybe it's a little more efficient (only when the user comes to the home page first), but we lose clarity.

For the differences between communites and footer icons, I have to admit it's quite blurry. I was thinking it's a place that pencil2d users can meet each others. Not directing users to another list of official announcements. But it's not a guideline just my personal opinions. We can discuss anything, I am open to that.

**Comment by J5lx** [comment]
For the record, I just updated the menu on the forum.

**Pull request merged**